src/main/java/com/microsoft/azure/util/AzureCredentials.java

```diff
@@ -686,19 +692,6 @@ public String getGraphEndpoint() {
         return data.graphEndpoint;
     }
 
-    @Override
-    public TokenCredentialData createToken() {
```

Comment:
Why remove this method?

Reply:
it didn't seem to be used anymore

Reply:
I have just checked this and this should be used in the Azure commons plugin. One sample class is https://github.com/jenkinsci/azure-commons-plugin/blob/dev/azure-commons-core/src/main/java/com/microsoft/jenkins/azurecommons/core/AzureClientFactory.java.

Reply:
I'll try test with the plugins and see if there's impact:
https://github.com/search?q=org%3Ajenkinsci+AzureClientFactory.getClient&type=Code

Still trying to resolve the other conflict, 😢

Reply:
This should be something with BI data.